include/wavm/OMPThreadPool.h

```diff
@@ -15,9 +15,6 @@
 #include <wavm/OMPThreadPool.h>
 
 namespace wasm {
-
-    using namespace WAVM;
```

Comment:
Unfortunately both WAVM and WAMR have classes/ structs named `HashMap` which conflict with each other, so I've had to be more explicit about using the WAVM namespace throughout the codebase.